wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -401,6 +404,27 @@ public RepeatCommand repeatedly() {
     return new RepeatCommand(this);
   }
 
+  /**
+   * Decorates this command to run repeatedly, restarting until the command runs for the given times
+   * amount. The decorated command can still be canceled.
+   *
+   * <p>Note: This decorator works by adding this command to a composition. The command the
+   * decorator was called on cannot be scheduled independently or be added to a different
+   * composition (namely, decorators), unless it is manually cleared from the list of composed
+   * commands with {@link CommandScheduler#removeComposedCommand(Command)}. The command composition
+   * returned from this method can be further decorated without issue.
+   *
+   * @param repetitions the count of times to run the command.
+   * @return the decorated command
+   */
+  public ParallelRaceGroup repeatedly(int repetitions) {
+    // use an array so that it stays in the heap instead of stack.
+    // We use an array with a size of 1 instead of `AtomicInteger` because of the performance
+    // overhead
+    int[] counter = new int[1];
```

Comment:
```suggestion
    int[] counter = { 0 };
```
This is more explicit about the initial value.